Source/SwiftLintFramework/Rules/Lint/NSObjectPreferIsEqualRule.swift

```diff
@@ -89,6 +83,14 @@ private extension FunctionDeclSyntax {
     }
 }
 
+private extension Syntax {
```

Comment:
I think if you extend SyntaxProtocol you don’t need to wrap with Syntax above.

Reply:
Good hint! See #4332.